api/BroadcastChannel.json

```diff
@@ -0,0 +1,277 @@
+{
+  "api": {
+    "BroadcastChannel": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/BroadcastChannel",
+        "support": {
+          "webview_android": {
+            "version_added": "54"
+          },
+          "chrome": {
+            "version_added": "54"
+          },
+          "chrome_android": {
+            "version_added": "54"
+          },
+          "firefox": {
+            "version_added": "38"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "41"
+          },
+          "opera_android": {
+            "version_added": "41"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "BroadcastChannel": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/BroadcastChannel/BroadcastChannel",
```

Comment:
Please add a description here: `"description": "<code>BroadcastChannel()</code> constructor",` (it might not be that obvious what a constructor is from looking at the data structure and this way the table should look like here: https://developer.mozilla.org//en-US/docs/Web/API/AbortController#Browser_compatibility)

See also https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables#Adding_a_description

Reply:
Fixed 👍
Thanks for the review